src/test/java/com/thangle/persistence/book/BookStoreTest.java

```diff
@@ -27,4 +28,44 @@ void shouldFindAll_OK() {
         assertEquals(expected.size(), bookStore.findAll().size());
         verify(bookRepository).findAll();
     }
+
+    @Test
+    void shouldCreate_OK() {
+        final var expected = buildBookEntity();
+        when(bookRepository.save(any())).thenReturn(expected);
+
+        final var actual = bookStore.create(buildBook());
+
+        assertEquals(actual.getId(), expected.getId());
```

Comment:
Please put expected as the first param of assert methods, and actual as the second one. Do this for all assertions